tests/Unit/Jobs/SendRemindersTest.php

```diff
@@ -38,33 +38,6 @@ public function test_it_schedules_a_reminder_email_job_and_a_set_next_expected_d
         Bus::assertDispatched(SetNextReminderDate::class);
     }
 
-    public function test_it_deletes_the_reminder_if_contact_doesnt_exist()
```

Comment:
This method should not be deleted. I also notice that the method name is wrong. But the method tests that the reminder is not sent if the contact does not exist (and not even dispatched) - which is a valuable test in my opinion.

Reply:
I removed it because it's not going to be technically possible anymore.

Once you delete a contact, due to the cascade constraints, the reminder will get deleted as well. So it's impossible for the contact of a reminder not to exist.

However this is because I set the contact_id to be mandatory (a constraint) for a reminder, maybe that isn't right?

Reply:
It is right, for now at least. So no need to deal with that for now on.